src/cleanAPICategoryTranslation.ts

```diff
@@ -0,0 +1,77 @@
+import { Translation } from './Types/Results';
+
+export const programTemplateCategories = [
+  'Housing and Utilities',
+  'Food and Nutrition',
+  'Health Care',
+  'Transportation',
+  'Tax Credits',
+  'Cash Assistance',
+  'Child Care, Youth, and Education',
+  'Food or Groceries',
+  'Baby Supplies',
+  'Managing housing costs',
+  'Behavioral health',
+  "Child's development",
+  'Family planning',
+  'Job resources',
+  'Low-cost dental care',
+  'Civil legal needs',
+];
+
+const alternateProgramTemplateCategories = [
+  {
+    previousNormalizedValues: ['diapers'],
+    correctValue: 'Baby Supplies',
+  },
+  {
+    previousNormalizedValues: ['housing'],
+    correctValue: 'Managing housing costs',
+  },
+  {
+    previousNormalizedValues: ['jobtraining'],
+    correctValue: 'Job resources',
+  },
+];
+
+const normalizeDefaultMessage = (defaultMessage: string): string => {
+  const whiteSpaceCommasApostropheOrHyphenMinusRegex = /\s|,|'|-/g;
```

Comment:
What do you think about removing any non-character or digit? That way we can cover more edge cases.
```suggestion
  const whiteSpaceCommasApostropheOrHyphenMinusRegex = /[^a-zA-Z0-9]/g;
```

Reply:
Good call! I've added that case to the regex pattern and updated the comments as necessary.